config/jobs/periodic/etcd/etcd-postsubmit.yaml

```diff
@@ -0,0 +1,47 @@
+postsubmits:
```

Comment:
why is this post submit? this is suppose to be periodic right?

Reply:
Hmm yes, my bad!
I copied the existing postsubmit job and edited it.
Have corrected it now. Please review now.